src/tests/ftest/aggregation/ec_aggregation_basic.py

```diff
@@ -0,0 +1,192 @@
+        total_runtime = self.params.get("total_runtime", 
+                                        '/run/aggregation_test/*')
+        pool_qty = total_pools
+        container_qty = total_containers_per_pool
+        start_time = 0
+        finish_time = 0
+        start_time = time.time()
+        while int(finish_time - start_time) < total_runtime:
+
+            job_manager = get_job_manager(self, "Mpirun", None, False, "mpich",
+                                          self.get_remaining_time())
+
+            # Create X pools
+            self.add_pool_qty(pool_qty, connect=False)
+            # Since the transfer size is 1M, the objects will be inserted
+            # directly into NVMe and hence storage_index = 1
+            storage_index = 1
+
+            initial_free_space = self.get_free_space(storage_index)
+
+            # Disable the aggregation
+            self.pool.set_property("reclaim", "disabled")
+
+            # Create Y containers per pool
+            for pool in self.pool:
+                self.add_container_qty(container_qty, pool)
+
+            # Run ior on each container sequentially
+            for container in self.container:
+                ior_log = "{}_{}_{}_ior.log".format(self.test_id,
+                                                    container.pool.uid,
+                                                    container.uuid)
+                try:
+                    self.ior_cmd.dfs_oclass.update(test_oclass)
+                    self.ior_cmd.dfs_dir_oclass.update(test_oclass)
+                    result = run_ior(self, job_manager, ior_log, self.hostlist_clients,
+                                     self.workdir, None, self.server_group, 
+                                     container.pool, container)
+                    self.log.info(result)
+                except CommandFailure as error:
+                    self.log.info(error)
+            free_space_after_first_ior = self.get_free_space(storage_index)
+            space_used_by_ior = initial_free_space - free_space_after_first_ior
+
+            self.log.info("Space used by first ior = %s", space_used_by_ior)
+            self.log.info("Free space after first ior = %s", free_space_after_first_ior)
+            self.assertTrue(free_space_after_first_ior < initial_free_space,
+                            "IOR run was not successful.")
+
+            # Run ior on each container sequentially
+            for container in self.container:
+                ior_log = "{}_{}_{}_ior.log".format(self.test_id, container.pool.uid, container.uuid)
+                try:
+                    self.ior_cmd.dfs_oclass.update(test_oclass)
+                    self.ior_cmd.dfs_dir_oclass.update(test_oclass)
+                    # Run ior the second time on the same pool and container, so another
+                    # copy of the file is inserted in DAOS.
+                    result = run_ior(self, job_manager, ior_log, self.hostlist_clients,
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'processes' in function call